doc/zephyr.doxyfile

```diff
@@ -406,7 +406,7 @@ LOOKUP_CACHE_SIZE      = 9
 # normally produced when WARNINGS is set to YES.
 # The default value is: NO.
 
-EXTRACT_ALL            = YES
+EXTRACT_ALL            = NO
```

Comment:
Hmm... there are lots of undocumented public members in the Zephyr public APIs that will disappear with this change?  Are you going to enforce developers providing doxygen comments on ALL public members now?  (We've had some issues raised because public members aren't showing up in the documentation already.)

Reply:
> Hmm... there are lots of undocumented public members in the Zephyr public APIs that will disappear with this change? 
If they really disappear, then we should get doc build errors/warnings, and we do not.

> Are you going to enforce developers providing doxygen comments on ALL public members now?

yes